### PR TITLE
For compatability, emit the \n within the library, not the cmd

### PR DESCRIPTION
**Pull request by sparkprime**

### CODE REVIEW

File: interpreter.go [hunk 1]
```diff
@@ -723,14 +723,14 @@ func serializeJSON(v interface{}, multiline bool, indent string, buf *bytes.Buff
 	}
 }
 
-func (i *interpreter) manifestAndSerializeJSON(trace *TraceElement, v value, multiline bool, indent string) (string, error) {
+func (i *interpreter) manifestAndSerializeJSON(trace *TraceElement, v value, multiline bool, indent string) (*bytes.Buffer, error) {
```
Comment (sbarzowski) [inline interpreter.go]: Maybe it would be better to take this buffer as an argument. Returning a buffer seems weird.

Comment (seh) [reply interpreter.go]: That, or even more generally, demand an `io.Writer`. The downside then is all the potential errors you have to deal with, whereas with a `bytes.Buffer` the operations cannot fail.

**Comment by sparkprime** [comment]
buf now passed in, and I added manifestString into the mix

**Comment by sbarzowski** [comment]
LGTM

**Pull request merged**